### PR TITLE
Bitrunner job figurine

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

This adds an action figure for Bitrunners, and places one in the Deltastation bitrunning den.

image

"I'm in..."
## Why It's Good For The Game

Every job has an action figure, and bitrunning should not be an exception! Delta has figurines in each job site, and adding one there lets the Bitrunners in on the fun.
## Changelog
:cl: Rhials
add: Bitrunners now have their own job figurine. Cool!
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1204,6 +1204,11 @@
 	icon_state = "bartender"
 	toysay = "Where is Pun Pun?"
 
+/obj/item/toy/figure/bitrunner
+	name = "\improper Bitrunner action figure"
+	icon_state = "bitrunner"
+	toysay = "Upgrades please!"
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm:1210]: should say "I'm in." instead

Comment (EEASAS) [reply code/game/objects/items/toys.dm:1210]: nuh uh

**Review by Wallemations — COMMENTED** [review]
> every job has a figurine

Have we added a coroner one yet?

**Comment by Rhials** [comment]
> Have we added a coroner one yet?

Yup, they already have one!
image
He looks like nostalgia critic if he had a blue beard

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Rhials) [reply code/game/objects/items/toys.dm:1210]: I'm going with this actually

**Pull request merged**